test/Java8andUp/playlist.xml

```diff
@@ -922,6 +922,69 @@
 		</subsets>
 	</test>
 
+	<test>
+		<featureIds>
+			<featureId>Java access control</featureId>
+		</featureIds>
+		<testCaseName>JCL_TEST_Java-Security_SE90</testCaseName>
```

Comment:
The final make target that gets generated takes testCaseName + Subset + NumberOfVariation (count starting from 0).  If you call the test this, the target name will be JCL_TEST_Java-Security_SE90_SE90_0  is that what you intend?  Same comment applies to testCaseName J9VMInternals_Test_SE80 below... J9VMInternals_Test_SE80_SE80_0 and J9VMInternals_Test_SE80_SE80_1,

Reply:
Shelley

In re your comment:

> the target name will be JCL_TEST_Java-Security_SE90_SE90_0

Could you please clarify?

Embedding the Java version appears to be not unusual in the playlist, cf. analogous existing entry <testCaseName>JCL_Test_SE90</testCaseName> (from which I extracted the new test cases), et al.

Looking at a recent internal build, I see:

make -f /bluebird/builds/bld_377668/jvmtest/test/SE90/TestConfig/wrapper.mk _JCL_Test_SE90_0 TEST_ROOT=...

and the generated makefile SE90/Java8andUp/autoGen.mk contains:

JCL_Test_SE90_0: TEST_RESROOT=$(JVM_TEST_ROOT)$(D)Java8andUp
JCL_Test_SE90_0: JVM_OPTIONS=$(RESERVED_OPTIONS) $(EXTRA_OPTIONS)
JCL_Test_SE90_0: TEST_GROUP=level.sanity
JCL_Test_SE90_0:
    @$(MKTREE) $(REPORTDIR);
    @echo "" | tee -a $(Q)$(TESTOUTPUT)$(D)TestTargetResult$(Q);
    @echo "===============================================" | tee -a $(Q)$(TESTOUTPUT)$(D)TestTargetResult$(Q);
    @echo "Running test $@ ..." | tee -a $(Q)$(TESTOUTPUT)$(D)TestTargetResult$(Q);
...



All that said, this is still WIP (and likely has other problems).
I'm trying to merge the two cases into one, so this may be moot.

@llxia @TianyuZuo FYI.

Reply:
Just to clarify, we do not need to add SE90 or SE80 in test name. However, the test name has to be unique. One case I have seen people use SE80 and SE90 is that if they have one test that uses different commands for SE80 and SE90, people use testXXX_SE80 and testXXX_SE90 as test name.  In your case, since the test will only run in SE90, there is no need to have SE90 in the test name.

Reply:
Correction: this test runs in both Java 8 and Java 9. As it happens, I believe I can use the same command line for both, so the point is now moot.  Personal build in progress.

Reply:
... or not:
  java.lang.AssertionError: unexpected: java.lang.reflect.InaccessibleObjectException: Unable to make field java.security.ProtectionDomain[] java.security.AccessControlContext.context accessible: module java.base does not "opens java.security" to unnamed module @6579a4e

Java 8 passes fine.

Looks like I need separate test cases with "--add-opens" for Java 9 and above.

Reply:
@smlambert @llxia please review.  
As note above, copyright failure is in the Java security policy file.

 Thanks.